src/gsi/read_avhrr_navy.f90

```diff
@@ -255,6 +255,7 @@ subroutine read_avhrr_navy(mype,val_avhrr,ithin,rmesh,jsatid,&
   next=0
 
 ! Read BUFR Navy data
+  nrec = 999999
```

Comment:
@wx20jjung , I forget ... why is array `nrec` initialized to `999999` in this and other readers?